src/neuroconv/tools/__init__.py

```diff
@@ -1,8 +1,7 @@
 # To be treated as sub-modules or plugins
-# TODO: temporarily disabled until decision to remove in other PR
-# from .spikeinterface import spikeinterface
-# from .roiextractors import roiextractors
-# from .neo import neo
+from .spikeinterface import spikeinterface
+from .roiextractors import roiextractors
+from .neo import neo
```

Comment:
@h-mayorquin This is why that import is still happening - when I said I tested locally, I meant on main (with the comments). So these should just be deleted, as in https://github.com/catalystneuro/neuroconv/pull/124)

May have to update the new import test to reflect this, however.